packages/explorer/src/observer/decorator.ts

```diff
@@ -37,6 +37,7 @@ export function observer({ explorerUrl = "http://localhost:13690", waitForTransa
       emit("write", {
         writeId,
         address: args.address,
+        from: client.account?.address,
```

Comment:
`writeContract` throws if no account exists
https://github.com/wevm/viem/blob/44ae1a5549e704486f2643c67f222ab3ce87cd47/src/actions/wallet/writeContract.ts#L184-L187

so I think we can just force this to be defined and not use an optional type here
```suggestion
        from: client.account!.address,
```

Reply:
cool was thinking to do the same, will force it